build.gradle.kts

```diff
@@ -42,29 +44,29 @@ dependencies {
     fun ktor(artifact: String) = "io.ktor:$artifact:$ktorVersion"
     fun sparetimedevs(artifact: String) = "com.sparetimedevs:$artifact:$sparetimedevsVersion"
 
-    compile(kotlin("kotlin-stdlib-jdk8"))
-
-    compile(kotlinx("kotlinx-coroutines-core"))
-    compile(kotlinx("kotlinx-coroutines-reactive"))
-
-    compile(ktor("ktor-server-netty"))
-    compile(ktor("ktor-html-builder"))
-    compile(ktor("ktor-jackson"))
-    compile(ktor("ktor-websockets"))
-    compile(ktor("ktor-metrics"))
-
-    compile(sparetimedevs("suspendmongo"))
-
-    compile("ch.qos.logback:logback-classic:$logbackVersion")
-    compile("org.slf4j:slf4j-api:$slf4jVersion")
-    compile("org.koin:koin-ktor:$koinVersion")
-    compile("io.github.cdimascio:java-dotenv:$javaDotEnvVersion")
+    implementation(kotlin("kotlin-stdlib-jdk8"))
+    implementation(kotlinx("kotlinx-coroutines-core"))
+    implementation(kotlinx("kotlinx-coroutines-reactive"))
+    implementation(ktor("ktor-server-netty"))
+    implementation(ktor("ktor-html-builder"))
+    implementation(ktor("ktor-jackson"))
+    implementation(ktor("ktor-websockets"))
+    implementation(ktor("ktor-metrics"))
+    implementation(ktor("ktor-locations"))
+    implementation(ktor("ktor-auth"))
+    implementation(ktor("ktor-auth-jwt"))
+
+    implementation(sparetimedevs("suspendmongo"))
+    implementation("ch.qos.logback:logback-classic:$logbackVersion")
+    implementation("org.slf4j:slf4j-api:$slf4jVersion")
+    implementation("org.koin:koin-ktor:$koinVersion")
+    implementation("io.github.cdimascio:java-dotenv:$javaDotEnvVersion")
+    implementation("com.auth0:java-jwt:$auth0JwtVersion")
+    implementation("io.github.microutils:kotlin-logging:$kotlinLoggingVersion")
 
     testCompile(kotlin("kotlin-test-junit"))
```

Comment:
Why not also change testCompile to testImplementation while you're at it? :)

Reply:
Logical indeed. Will do so.